contrib/devtools/update-translations.py

```diff
@@ -160,7 +166,7 @@ def postprocess_translations(reduce_diff_hacks=False):
                     if translation is None:
                         continue
                     errors = []
-                    valid = check_format_specifiers(source, translation, errors, numerus)
+                    valid = check_format_specifiers(source, translation, errors, numerus) and not contains_bitcoin_addr(translation_node.text)
```

Comment:
Should say `translation` instead of `translation_node`?

Reply:
Whoops! Good catch. Copied and pasted from my previous commit. Updating now